packages/opal-common/opal_common/paths.py

```diff
@@ -87,10 +87,13 @@ def glob_style_match_path_to_list(path: str, match_paths: List[str]):
         """
         # check if any of our ignore paths match the given path
         for match_path in match_paths:
+            # if the path is the root "/", then it matches any path
+            if match_path == "/" or match_path == "/**":
+                return match_path
             # if the path is indicated as a parent via "/**" at the end
             if match_path.endswith("/**"):
                 # check if the path is under the parent
-                if path.startswith(match_path[:-3]):
+                if (path + "/").startswith((match_path[:-3] + "/")):
```

Comment:
This change fixes a bug generating false positive results. For example it occurs with values as `match_path == myFolder` and `path == myFolderTwo`